packages/core/src/database/postgres/service.test.ts

```diff
@@ -55,6 +56,16 @@ const schemaTwo = createSchema((p) => ({
   }),
 }));
 
+const schemaWithDefaults = createSchema((p) => ({
+  Dog: p.createTable({
+    id: p.string().defaultTo("0"),
+    name: p.string().defaultTo("firstname"),
+    age: p.int().defaultTo(5).optional(),
+    bigAge: p.bigint().defaultTo(5n).optional(),
```

Comment:
Couple general notes:
- Per the discussion [here](https://github.com/ponder-sh/ponder/issues/975) I propose we go with `.default()` over `.defaultTo()`
- I'm not sure if this is handled yet, but adding `.default` to a column should automatically/implicitly make it optional. Perhaps we should validate against including both `.optional()` and `.default()` to a column?

Reply:
ah, thanks for linking. forgot to do that. regarding your points:
* if we do not mind the js `.default` export pattern and wish to distinguish, i am ok with that
* I don't know that it should actually. Optional only denotes nullability, which should still be possible to do even if you started with a non null value. In other words, when I first insert, I don't have a value, the database will use the default value for me, but I should be able to turn that into null later on or use null during insertion.

Reply:
Re 1: I think it's fine - this is being used in a context rather distant from a module export. Fwiw Drizzle uses [`.default`](https://orm.drizzle.team/docs/column-types/pg#default-value) as well.

Re 2: Ah yes, my mistake - you're right. The interesting case is: if a column has a default and is NOT optional, it should still be optional when inserting or updating records of that table (but not when selecting). I'm not sure how well our existing type system is set up to handle this, cc @kyscott18

Reply:
💯

Reply:
should (probably) be addressable in this pr